xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+      already includes <systemitem>nova-api</systemitem>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Commit your changes to the git model:
+     </para>
+     <screen>(deployer)> git commit -a -m "Prepare for HOS 5.1 installation"</screen>
+    </listitem>
+   </itemizedlist>
+  </listitem>
+  <listitem>
+   <para>
+    Consider the effect of control plane downtime on existing DHCP leases:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Default DHCP lease time is 48 hours as defined in the
+      <filename>~/helion/hos/ansible/roles/neutron-common/templates/neutron.conf.j2</filename> file by the
+      <systemitem>dhcp_lease_duration</systemitem> parameter.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP leases are administered by &o_netw; and dnsmasq. This functionality will not be available
+      during the control plane downtime.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP clients typically renew their leases either 50% of the way into the lease time or upon
+      reboot.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Normally, this leaves a 24-hour window for control
+      plane downtime at any point before workload's DHCP leases run out and
+      cannot be renewed. This is enough for performing a migration.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      To view the upcoming timeouts of DHCP leases, run the following command from a &contrnode;:
+     </para>
+     <screen>for i in $(cat /var/run/neutron/dhcp/*/leases | awk '{print $1}'); do date -d @$i; done</screen>
+    </listitem>
+    <listitem>
+     <para>
+      If you find it necessary to increase the DHCP lease time before
+      migrating, open the
+      <filename>~/helion/hos/ansible/roles/neutron-common/templates/neutron.conf.j2</filename>
+      and edit the <systemitem>dhcp_lease_duration</systemitem> parameter. 
+      Commit the change, and run the <filename>neutron-configure.yml</filename> playbook.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </listitem>
+  </itemizedlist>
+</step>
+<step>
+ <para>
+  Decommission the seed node from the existing cloud (if the intended seed node hosts control plane
+  services beyond the database):
+  <remark role="clarity">taroth 2018-02-27: is 'if the intended seed node hosts control plane
+  services beyond the database' the condition which must be fulfilled to execute
+  the following two actions (in the itemizedlist)? if yes, rephrase the sentence to: 
+  'If the intended seed node hosts control plane services beyond the database, decommission the seed
+   node from the existing cloud.' If the following two actions explain how to decommission, end the
+   introductory sentence with a colon: 'If the intended seed node hosts control plane services beyond
+   the database, decommission the seed node from the existing cloud:'</remark>
```

Comment:
My impression is that the two following list items are not explaining decommissioning per se, they just indicate caveats. So, maybe add a sentence like "Keep in mind the following while decommisioning:" at the end of the para.